### PR TITLE
[Docs] Replace the models used in the tutorial document with RTMDet

### PR DESCRIPTION
**Pull request by Zheng-LinXiao**
Thanks for your contribution and we appreciate it a lot. The following instructions would make your pull request more healthy and more easily get feedback. If you do not understand some items, don't worry, just make the pull request and seek help from maintainers.

## Motivation

Please describe the motivation of this PR and the goal you want to achieve through this PR.

## Modification

Please briefly describe what modification is made in this PR.

## BC-breaking (Optional)

Does the modification introduce changes that break the backward-compatibility of the downstream repos?
If so, please describe how it breaks the compatibility and how the downstream projects should modify their code to keep compatibility with this PR.

## Use cases (Optional)

If this PR introduces a new feature, it is better to list some use cases here, and update the documentation.

## Checklist

1. Pre-commit or other linting tools are used to fix the potential lint issues.
2. The modification is covered by complete unit tests. If not, please add more unit test to ensure the correctness.
3. If the modification has potential influence on downstream projects, this PR should be tested with downstream projects, like MMDet or MMCls.
4. The documentation has been modified accordingly, like docstring or example tutorials.

### CODE REVIEW

File: docs/en/user_guides/config.md [hunk 1]
```diff
@@ -1,177 +1,87 @@
...
+MMDetection uses a modular design, all modules with different functions can be configured through the config. Taking RTMDet as an example, we will introduce each field in the config according to different function modules:
 
 ### Model config
 
 In MMDetection's config, we use `model` to set up detection algorithm components. In addition to neural network components such as `backbone`, `neck`, etc, it also requires `data_preprocessor`, `train_cfg`, and `test_cfg`. `data_preprocessor` is responsible for processing a batch of data output by dataloader. `train_cfg`, and `test_cfg` in the model config are for training and testing hyperparameters of the components.
 
 ```python
 model = dict(
-    type='MaskRCNN',  # The name of detector
+    type='RTMDet',  # The name of detector
```
Comment (BIGWangYuDong) [inline docs/en/user_guides/config.md:15]: maybe we can keep MaskRCNN, and add RTMDet config somewhere

File: docs/en/user_guides/test.md [hunk 2]
```diff
@@ -54,24 +54,24 @@ Optional arguments:
...
-   Config and checkpoint files are available [here](../../../configs/faster_rcnn).
+1. Test RTMDet and visualize the results. Press any key for the next image.
+   Config and checkpoint files are available [here](https://github.com/open-mmlab/mmdetection/tree/3.x/configs/rtmdet).
 
    ```shell
    python tools/test.py \
-       configs/faster_rcnn/faster-rcnn_r50_fpn_1x_coco.py \
-       checkpoints/faster_rcnn_r50_fpn_1x_coco_20200130-047c8118.pth \
-       --show
+        configs/rtmdet/rtmdet_l_8xb32-300e_coco.py \
```
Comment (BIGWangYuDong) [inline docs/en/user_guides/test.md:62]: unnecessary space adding

File: docs/en/user_guides/test.md [hunk 3]
```diff
@@ -54,24 +54,24 @@ Optional arguments:
...
-   Config and checkpoint files are available [here](../../../configs/faster_rcnn).
+2. Test RTMDet and save the painted images for future visualization.
+   Config and checkpoint files are available [here](https://github.com/open-mmlab/mmdetection/tree/3.x/configs/rtmdet).
 
    ```shell
    python tools/test.py \
-       configs/faster_rcnn/faster-rcnn_r50_fpn_1x_coco.py \
-       checkpoints/faster_rcnn_r50_fpn_1x_coco_20200130-047c8118.pth \
-       --show-dir faster_rcnn_r50_fpn_1x_results
+        configs/rtmdet/rtmdet_l_8xb32-300e_coco.py \
```
Comment (BIGWangYuDong) [inline docs/en/user_guides/test.md:72]: unnecessary space adding

File: docs/en/user_guides/train.md [hunk 4]
```diff
@@ -13,7 +13,7 @@ You could download the existing models in advance if the network connection is u
 
 ## Learning rate auto scaling
 
-**Important**: The default learning rate in config files is for 8 GPUs and 2 sample per GPU (batch size = 8 * 2 = 16). And it had been set to `auto_scale_lr.base_batch_size` in `config/_base_/default_runtime.py`. Learning rate will be automatically scaled base on this value when the batch size is `16`. Meanwhile, in order not to affect other codebase which based on mmdet, the flag `auto_scale_lr.enable` is set to `False` by default.
+**Important**: The default learning rate in config files is for 8 GPUs and 2 sample per GPU (batch size = 8 * 2 = 16). And it had been set to `auto_scale_lr.base_batch_size` in `config/_base_/schedules/schedule_xx.py`. Learning rate will be automatically scaled base on this value when the batch size is `16`. Meanwhile, in order not to affect other codebase which based on mmdet, the flag `auto_scale_lr.enable` is set to `False` by default.
```
Comment (BIGWangYuDong) [inline docs/en/user_guides/train.md:16]: schedule_xx  can change to schedule_1x

File: docs/zh_cn/get_started.md [hunk 5]
```diff
@@ -95,8 +95,8 @@ python demo/image_demo.py demo/demo.jpg yolov3_mobilenetv2_8xb24-320-300e_coco.p
 ```python
 from mmdet.apis import init_detector, inference_detector
 
-config_file = 'yolov3_mobilenetv2_8xb24-320-300e_coco.py'
-checkpoint_file = 'yolov3_mobilenetv2_320_300e_coco_20210719_215349-d18dff72.pth'
```
Comment (RangiLyu) [inline docs/zh_cn/get_started.md:99]: yolov3_mobilenetv2 是个很小的模型，这里建议换成 RTMDet-tiny

File: docs/en/user_guides/train.md [hunk 6]
```diff
@@ -385,16 +385,16 @@ Using the function above, users can successfully convert the annotation file int
 
 ## Prepare a config
 
-The second step is to prepare a config thus the dataset could be successfully loaded. Assume that we want to use Mask R-CNN with FPN, the config to train the detector on balloon dataset is as below. Assume the config is under directory `configs/balloon/` and named as `mask-rcnn_r50-caffe_fpn_ms-poly-1x_balloon.py`, the config is as below.
+The second step is to prepare a config thus the dataset could be successfully loaded. Assume that we want to use RTMDet, the config to train the detector on balloon dataset is as below. Assume the config is under directory `configs/balloon/` and named as `rtmdet_l_8xb32-300e_balloon.py`, the config is as below.
```
Comment (RangiLyu) [inline docs/en/user_guides/train.md:388]: 这里的例子建议保持使用 Mask R-CNN，因为这个数据集是实例分割数据集

File: docs/zh_cn/user_guides/config.md [hunk 7]
```diff
@@ -1,203 +1,113 @@
 # 学习配置文件
 
-MMDetection 和其他 OpenMMLab 仓库使用 [MMEngine 的配置文件系统](https://mmengine.readthedocs.io/zh_cn/latest/tutorials/config.md)。 配置文件使用了模块化和继承设计，以便于进行各类实验。
+MMDetection 和其他 OpenMMLab 仓库使用 [MMEngine 的配置文件系统](https://mmengine.readthedocs.io/zh_CN/latest/advanced_tutorials/config.html) 。配置文件使用了模块化和继承设计，以便于进行各类实验。
 
 ## 配置文件的内容
 
-MMDetection 采用模块化设计，所有功能的模块都可以通过配置文件进行配置。 以 Mask R-CNN 为例，我们将根据不同的功能模块介绍配置文件中的各个字段：
+MMDetection 采用模块化设计，所有功能的模块都可以通过配置文件进行配置。 以 RTMDet 为例，我们将根据不同的功能模块介绍配置文件中的各个字段：
```
Comment (RangiLyu) [inline docs/zh_cn/user_guides/config.md:7]: 建议换回 Mask R-CNN 作为配置文件说明，Mask R-CNN 配置更全，而且是标准的1x schedule，适用于mmdet的绝大部分其他模型。

File: docs/zh_cn/user_guides/train.md [hunk 8]
```diff
@@ -387,16 +387,16 @@ if __name__ == '__main__':
 
 ## 准备配置文件
 
-第二步需要准备一个配置文件来成功加载数据集。假设我们想要用 balloon dataset 来训练配备了 FPN 的 Mask R-CNN ，如下是我们的配置文件。假设配置文件命名为 `mask-rcnn_r50-caffe_fpn_ms-poly-1x_balloon.py`，相应保存路径为 `configs/balloon/`，配置文件内容如下所示。
+第二步需要准备一个配置文件来成功加载数据集。假设我们想要用 balloon dataset 来训练 RTMDet ，如下是我们的配置文件。假设配置文件命名为 `rtmdet_l_8xb32-300e_balloon.py`，相应保存路径为 `configs/balloon/`，配置文件内容如下所示。
```
Comment (RangiLyu) [inline docs/zh_cn/user_guides/train.md:390]: 涉及到训练的教程都换回 Mask R-CNN

Comment (Zheng-LinXiao) [reply docs/zh_cn/user_guides/train.md:390]: 收到

File: docs/zh_cn/get_started.md (hunk 5 above)
Comment (Zheng-LinXiao) [reply docs/zh_cn/get_started.md:99]: 好的

**Review by RangiLyu — APPROVED** [review]

**Review by BIGWangYuDong — APPROVED** [review]

File: docs/zh_cn/user_guides/config.md (hunk 7 above)
Comment (ZwwWayne) [reply docs/zh_cn/user_guides/config.md]: We can put these contents in the README.md of rtmdet
